scripts/redirect_url_utils.py

```diff
@@ -1,14 +1,45 @@
 #!/usr/bin/env python3
 
-# To create redirect files from stored map run
-# cd okta-help
-# python3 scripts/redirect_url_utils.py apply_map
-# or simply
-# python3 scripts/redirect_url_utils.py
-#
-# To update file paths to redirect urls map run
-# cd okta-help
-# python3 scripts/redirect_url_utils.py save_map
+"""
+
+Applying existing redirects
+------------------
+
+The default action of this script is to write all existing redirects in
+`file_to_url.txt` to the file system. At present, that's over 8,000 files.
+
+        $ python scripts/redirect_url_utils.py
```

Comment:
Does this instruction work for you? I'm receiving an error
<img width="905" alt="image" src="https://github.com/okta/okta-help/assets/4158731/3ff41555-729f-4cb6-a369-0a1f3c228a3d">